TPL/force-app/main/default/flows/Update_Healthcare_Cost_Fields.flow-meta.xml

```diff
@@ -127,6 +127,13 @@ DateId</description>
                 <elementReference>dateidformula</elementReference>
             </value>
         </assignmentItems>
+        <assignmentItems>
```

Comment:
I requested the syncing of Date of Service with Admission Date on hospital costs so that Date of Service can be hidden from view if requested by the client. This is needed as the sortation reference dateid is keyed to date of service.